Sources/Logging/Logging.swift

```diff
@@ -255,7 +261,6 @@ extension Logger {
     ///    - line: The line this log message originates from (there's usually no need to pass it explicitly as it
     ///            defaults to `#line`).
     #if compiler(>=5.3)
-    @inlinable
```

Comment:
Why did this `@inlinable` get removed?

Reply:
oh no, it should not 😱 I will do another clear pass...